tests/Unit/PointwiseFunctions/Hydro/EquationsOfState/Python/Test_Enthalpy.py

```diff
@@ -0,0 +1,136 @@
+# Distributed under the MIT License.
+# See LICENSE.txt for details.
+
+import json
+import unittest
+
+import EnthalpyEos as EnthalpyPY
+import numpy as np
+import numpy.testing as npt
+import pandas as pd
+import scipy.integrate as integrate
+import scipy.interpolate as interpolate
+import SpectralEos as SpectralPY
+
+import spectre
+import spectre.PointwiseFunctions.Hydro.EquationsOfState as EoS
+from spectre.DataStructures import DataVector
+from spectre.DataStructures.Tensor import Scalar
+
+
+class TestEnthalpy(unittest.TestCase):
+    def test_enthalpy(self):
+        ##Set parameters:
+
+        low_eos_reference_density = 4.533804669935759e-05
+        low_eos_reference_pressure = 9.970647727158039e-08
+        low_eos_spectral_coefficients = [
+            1.2,
+            0.0,
+            1.34440187653529,
+            -0.46098357752567365,
+        ]
+        low_eos_upper_density = 0.0004533804669935759
+
+        reference_density = 0.00022669023349678794
+        max_density = 0.0031736632689550316
+        min_density = low_eos_upper_density
+        trig_scale = 1.26426988871305
+        polynomial_coefficients = [
+            1.0,
+            0.08063293075870805,
+            4.406887319408924e-26,
+            8.177895241388924e-22,
+            0.013558242085066733,
+            0.004117320982626606,
+            9.757362504479485e-26,
+            1.5646573325075753e-30,
+            0.00016253964205058317,
+        ]
+        sin_coefficients = [
+            0.0003763514388305583,
+            0.017968749910748837,
+            0.008140052979970034,
+            -0.003067418379116628,
+            -0.0008236601907322793,
+        ]
+        cos_coefficients = [
+            -0.01080996024705052,
+            -0.003421193490191067,
+            0.012325774692378716,
+            0.004367136076912163,
+            -0.00020374276952538073,
+        ]
+        transition_delta_epsilon = 0.0
+
+        ## Low Density Equation:
+
+        # Generate python spectral equation of state.
```

Comment:
Is the idea here to re-implement the EOS in Python and compare to the C++ implementation? Isn't the EOS implementation tested sufficiently in C++ already? I think here in Python you only have to check that the bindings you wrote work, meaning that you can construct and invoke the EOS in Python. That's why the PolytropicFluid EOS has the comment "Only testing p(rho) because the class is tested thoroughly in C++". It's important that unit tests only test the particular unit of code they are written for (the bindings in this case, not the full EOS implementation). Otherwise the unit tests become harder to maintain. Therefore, I suggest you simplify these Python tests significantly. You could call the Python re-implementation in the C++ test if you think it would strengthen the test.

Reply:
Ok then I suggest you call the EOS from the C++ test and keep the Python test very simple (because the code it tests is very simple: it's just the binding code).